src/webgl/p5.RenderBuffer.js

```diff
@@ -8,6 +8,28 @@ p5.RenderBuffer = class {
     this.attr = attr; // the name of the vertex attribute
     this._renderer = renderer;
     this.map = map; // optional, a transformation function to apply to src
+    this._namespace = undefined;
+    this._stride = 0;
+    this._offset = 0;
+    this._divisor = undefined;
+  }
+  namespace(namespace) {
```

Comment:
Why wouldn't we just add these properties to the constructor instead of setting them through chaining functions during construction?

Reply:
Mostly because it gets hard to read with a long chain of parameters, and because it's valid to have some of these without others and having `undefined` in the middle of the list feels weird. I could also refactor this to use something like an options object for the constructor, which could serve a similar purpose?

Reply:
Yeah, I think an options object sounds like it would be much cleaner here if you're open to doing that refactor.